app/policies/plan_policy.rb

```diff
@@ -0,0 +1,16 @@
+class PlanPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+  permit_admin_to :new, :create, :edit, :update, :destroy
+
+  def show?
+    true
+  end
```

Comment:
Because we are permitting specific actions to admin only. To permit action to all users we have to define action policy specifically. The default policy is to return `false`.